.github/workflows/ci-daily.yml

```diff
@@ -1,4 +1,4 @@
-name: CI - FAST
+name: Validate
```

Comment:
Is Validate clear enough? Is this just validating no breaks on the main branch?

Reply:
I believe so, and yes.